### PR TITLE
[Web Application] - sql connect application to database

### LINKED ISSUE #42: [Web Application] Create Database and connect web application
As a recurring driver, I want the web application to be connected to a database, so that my data is saved for later use to give a better experience.

### PR DESCRIPTION
**Pull request by ma-esf**
The application automatically creates and maintains the database on load

### CODE REVIEW

**Comment by leithhobson** [comment]
This PR resolves the majority of #42

**Review by diaistanbuly — APPROVED** [review]
#52 resolved the basic requirements for the database

File: Server/online.quar/src/main/java/online/quar/application/manager/DatabaseManager.java [hunk 1]
```diff
@@ -40,13 +44,28 @@ public void initialise() throws SQLException {
...
+            log.c(e.getMessage());
         }
 
-        rs.close();
+      //  ResultSet rs = executeQuery(c, "SELECT 15 AS retval;");
+//        if(rs.next()) {
+//
+//        }
+//
+//        rs.close();
```
Comment (leithhobson) [inline Server/online.quar/src/main/java/online/quar/application/manager/DatabaseManager.java:66]: Remember to clean up :)

**Review by leithhobson — APPROVED** [review]
Looks good overall

**Pull request merged**